hc-bar/js/bar.js

```diff
@@ -0,0 +1,314 @@
+      dataObject[eventRow][drilldownRow].y += 1;
+
+      // populate drilldown series
+      if (!drilldownObject[drilldownRow]) {
+        drilldownObject[drilldownRow] = {
+          name: eventRow,
+          id: drilldownRow,
+          data: [],
+          yAxis: 1,
+          pointWidth: 20,
+          pointRange: 1
+        };
+      }
+      // Populate drilldown data
+      drilldownObject[drilldownRow].data.push({
+        x: drilldownDate,
+        y: 1,
+        toolHeader: row[9],
+        toolText: eventText
+      });
+    });
+    // Correct formatting
+    // Create array of event types
+    var seriesNames = Object.keys(dataObject);
+    // Create arrays of the data for each event
+    dataArray = Object.values(dataObject);
+    for (var i = 0; i < dataArray.length; i++) {
+      // Remove extra object level
+      var dataPoints = Object.values(dataArray[i]);
+      // Get series color based on event type
+      var eventColor = "";
+      if (seriesNames[i] == "intercept") {
+        eventColor = "#9acd32";
+      } else if (seriesNames[i] == "strike") {
+        eventColor = "#954950";
+      } else {
+        eventColor = "#BEBDC0";
+      }
+      // Push event name and reformatted data objects to series
+      series.push({
+        name: seriesNames[i],
+        data: dataPoints,
+        xAxis: 0,
+        yAxis: 0,
+        color: eventColor
+      });
+    }
+    // Remove extra object level
+    drilldown = Object.values(drilldownObject);
+    renderChart(series, drilldown);
+  }
+});
+
+function renderChart(series, drilldown) {
+  // Remove link in x axis titles
+  Highcharts.Tick.prototype.drillable = function () { }
+
+  // format drillup button
+  Highcharts.setOptions({
+    lang: {
+      drillUpText: "◁ Back to Main"
+    },
+    global: {
+      useUTC: false
+    },
+  });
+  Highcharts.chart("hcContainer", {
+    // General Chart Options
+    chart: {
+      type: "column",
+      events: {
+        load: function () {
+          let top = this.plotTop + this.plotHeight + 92.5,
+            left = this.plotLeft + this.plotWidth - 150
+          image = this.renderer.image('https://missilethreat.csis.org/hc-bar/MissileDefense.png', left, top, 150, 15)
+          image.add()
+        },
+        redraw: function () {
+          let top = this.plotTop + this.plotHeight + 92.5,
+            left = this.plotLeft + this.plotWidth - 150
+          image.attr({ x: left, y: top })
+        },
+        // On drilldown remove yAxis title and format x axis labels
+        drilldown: function (e) {
+          this.yAxis[0].setTitle({ text: undefined })
+          const date = new Date(e.category)
+          let firstDay = new Date(date.getFullYear(), date.getMonth(), 1)
+          let lastDay = new Date(date.getFullYear(), date.getMonth() + 1, 0)
+          this.xAxis[0].update({
+            labels: {
+              format: "{value:%e}",
```

Comment:
Consistency is always great in code. This codebase is in ES5, so its super fascinating that once we get into the HighCharts.Chart section ES6 code is allowed to be compiled. For consistency's sake, lets stick to the ES5 syntax that we have so far. Lets change the lets to vars that we have in this object that raps the remaining codebase.